package.json

```diff
@@ -69,7 +69,7 @@
     "eslint-plugin-html": "^6.1.1",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsdoc": "^32.3.4",
-    "flow-bin": "^0.100.0",
+    "flow-bin": "0.103.0",
```

Comment:
The old flow version didn't support flatMap(). This is no longer relevant for this P.R. but should be helpful in the future.

Reply:
I'm not opposed to it, but it seems like this has a very large effect on `package.lock`. I haven't worked through the diff of that file, so just making sure: does it seem like the changes are within reason?

<img width="446" alt="Screen Shot 2021-08-17 at 10 12 16 AM" src="https://user-images.githubusercontent.com/572717/129770830-cb943a61-ffbb-47c0-a479-3cef083b32d4.png">

Reply:
(Like maybe it's just reorganization or indentation changes and not actually 12000 different npm modules we're loading?)

Reply:
Giving it a quick skim, all I'm seeing is reordering, indentation and removing quotation marks. Is there a diffing tool I could use to capture actual changes without so much noise?

Reply:
Maybe this: https://www.npmjs.com/package/yarn-lock-diff

Reply:
@rreusser Here we are:
│ package name                 │ old version(s)                                   │ new version(s)      
│ async                        │ [..., 2.6.2]                                     │ [...]               
│ caching-transform            │ 3.0.1                                            │ 3.0.2               
│ find-cache-dir               │ 2.0.0                                            │ 2.1.0               
│ flow-bin                     │ 0.100.0                                          │ 0.103.0             
│ handlebars                   │ [..., 4.7.7], 4.1.0                              │ [...], 4.7.7        
│ istanbul-lib-coverage        │ [..., 3.0.0], 2.0.5, 2.0.3                       │ [...], 3.0.0, 2.0.5 
│ istanbul-lib-hook            │ 2.0.3                                            │ 2.0.7               
│ istanbul-lib-report          │ 2.0.4                                            │ 2.0.8               
│ istanbul-lib-source-maps     │ 3.0.2                                            │ 3.0.6               
│ istanbul-reports             │ 2.1.1                                            │ 2.2.7               
│ lodash                       │ [..., 4.17.21], 4.17.11                          │ [...], 4.17.21      
│ make-dir                     │ [...], 3.1.0                                     │ [..., 3.1.0], 2.1.0 
│ mem                          │ 4.1.0                                            │ 4.3.0               
│ mimic-fn                     │ 1.2.0                                            │ 2.1.0               
│ minimist                     │ [..., 1.2.5], 1.2.0, 0.0.8                       │ [...], 1.2.5, 1.2.0 
│ mkdirp                       │ [..., 1.0.4], 0.5.5, 0.5.1                       │ [...], 1.0.4, 0.5.5 
│ optimist                     │ 0.6.1                                            │ -                   
│ p-is-promise                 │ 2.0.0                                            │ 2.1.0               
│ pify                         │ [...], 5.0.0                                     │ [..., 5.0.0], 4.0.1 
│ rimraf                       │ [..., 3.0.2], 2.7.1, 2.6.3                       │ [...], 3.0.2, 2.7.1 
│ spawn-wrap                   │ 1.4.2                                            │ 1.4.3               
│ test-exclude                 │ 5.1.0                                            │ 5.2.3               
│ y18n                         │ 4.0.1, 4.0.0                                     │ 4.0.3, 4.0.1